js/toolbar.js

```diff
@@ -322,10 +322,10 @@ class Toolbar {
             handleClick();
             stopIcon.style.color = this.stopIconColorWhenPlaying;
             isPlayIconRunning = true; 
-            variable= setTimeout(function() { handleClick(); }, 4600);
+            timeVar = setTimeout(function() { handleClick(); }, 4600);
```

Comment:
Does it take 4600 ms to start to play? Or to finish playing? We want to debounce at the start, but we cannot wait for every project to finish. I would think 2000 ms is more than enough.

Reply:
I push it for 2000 ms. I think it's fine for 2000.